fortanix-vme/nsm/src/lib.rs

```diff
@@ -0,0 +1,263 @@
+//! The nsm_io library provides an interface to the AWS Nitro Security Module (nsm). Unfortunately,
+//! the interface of this crate is not very Rust friendly. To avoid clients of this crate having to
+//! write the same wrapper code over and over again, this crate does this once.
```

Comment:
Looks like the term "this crate" is used both to refer to the `nsm_io` library and to this `nsm` crate.